spi.h

```diff
@@ -26,6 +26,7 @@
 #define C_SPR0 1
 
 void spi_init();
+void spi_reconfigure(uint8_t cpol, uint8_t cpha);
```

Comment:
I would call it something like `spi_set_mode`, since the combination of `cpol` and `cpha` is defined as "mode" in the SPI standard.